src/server/sql/logmsg/get_logs_from_dates_and_type.sql

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+-- Gets logs in table by date range. This is then ordered by time ascending.
+SELECT 
+	log_type, log_message, log_time
+FROM logmsg 
+WHERE log_type = ${logType}
+	AND log_time >= COALESCE(${startDate}, '-infinity'::TIMESTAMP)
```

Comment:
Just a note that we need to think about allowing long time ranges and defaulting to infinity. This, indirectly, applies to getting them all and some others too.